dbt/adapters/bigquery/impl.py

```diff
@@ -907,3 +909,26 @@ def python_submission_helpers(self) -> Dict[str, Type[PythonJobHelper]]:
             "cluster": ClusterDataprocHelper,
             "serverless": ServerlessDataProcHelper,
         }
+
+    @classmethod
+    def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
+        if constraint.type == ConstraintType.not_null:
+            return super().render_column_constraint(constraint)
+        elif constraint.type == ConstraintType.primary_key:
+            return "primary key not enforced"
+        elif constraint.type == ConstraintType.foreign_key:
+            c = super().render_column_constraint(constraint)
+            return f"{c} not enforced"
+        else:
+            return ""
```

Comment:
Any particular reason this method returns an empty string while the method below returns a `None`? What's the contract of the upstream class?